docs/how-to-guides/creating-grc721.md

```diff
@@ -0,0 +1,180 @@
+---
+id: creating-grc721
+---
+
+# How to create a GRC721 Token (NFT)
+
+## Overview
+
+This guide shows you how to write a simple _GRC721_ Smart Contract, or rather a [Realm](../explanation/realms.md), in [Gnolang (Gno)](../explanation/gno-language.md). For actually deploying the Realm, please see the [deployment](deploy.md) guide.
+
+Our _GRC721_ Realm will have the following functionality:
+
+- Minting a configurable amount of token.
+- Keeping track of total token supply.
+- Fetching the balance of an account.
+
+## Prerequisites
+
+We will proceed using the typical directory structure for a Realm found within the [simple-contract guide](simple-contract.md). It is also worthwhile to consult the [GRC721 interface](../../examples/gno.land/p/demo/grc/grc721/igrc721.gno) which we will be borrowing from within this guide.
+
+## 1. Importing token package
+For this realm, we'll want to import the `grc20` package as this will include the main functionality of our token factory realm.
+
+mynonfungibletoken.gno
+```go
+package mynonfungibletoken
+
+import (
+    "std"
+
+	"gno.land/p/demo/grc/grc721"
+	"gno.land/p/demo/ufmt"
+	"gno.land/r/demo/users"
+)
+
+var (
+	admin std.Address  = "g1us8428u2a5satrlxzagqqa5m6vmuze025anjlj" // set admin account
+    // provision the token's name and symbol
+    mynonfungibletoken = grc721.NewBasicNFT("mynonfungibletoken", "MNFT")
```

Comment:
Mixing tabs and spaces can cause issues.

By the way, this problem can also be resolved using `embedmd`.